tools/README.md

```diff
@@ -8,7 +8,11 @@
 
 ## Vim plugins
 
-- ...
+- [syntastic](https://github.com/scrooloose/syntastic) - syntax checking plugin
```

Comment:
Paki lagyan na lang siguro ng sections, para madaling dagdagan ng alternatives.

And mag `nvim` ka na @marksteve. Onti lang tatamaan, kung meron.